src/lashup_gm.erl

```diff
@@ -445,7 +434,42 @@ persist(Member, _State) ->
       lashup_gm_events:ingest(OldMember, Member);
     [] ->
       ets:insert(members, Member),
+      prometheus_gauge:set(
+        lashup, gm_members, [],
+        ets:info(members, size)),
       lashup_gm_events:ingest(Member)
   end,
   %% Find the component I'm part of
   ok.
+
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+  prometheus_gauge:new([
+    {registry, lashup},
+    {name, gm_members},
+    {help, "The size of global membership table."}
+  ]),
+  prometheus_gauge:new([
+    {registry, lashup},
+    {name, gm_subscriptions},
+    {help, "The number of global membership subscriptions."}
+  ]),
+  prometheus_gauge:new([
+    {registry, lashup},
+    {name, gm_subscribers},
+    {help, "The number of global membership subscribers."}
+  ]),
+  prometheus_counter:new([
+    {registry, lashup},
+    {name, gm_updates_total},
+    {help, "Total number of global membership table updates."}
```

Comment:
nit: I guess, it should start with `The` since the other ones do :)

Reply:
It was intentional. I checked several Erlang projects and almost all of them have "Total number of" for counter metrics, but "The time/bytes/size/lenght/etc" for other metric types. I've check it once again. 70% projects on github (all languages, mostly go) have prometheus metrics with "Total number of" help string and only 30% have metrics with help that starts with "The total number".